buckw

```diff
@@ -213,8 +213,8 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
+    # Go directly to kill or --help. Do not run okbuck.
+    if [[ "kill" == $1 || "--help" == $2 ]]; then
```

Comment:
Good point, `buck --help` is an option. I've only used `buck <command> --help`. I also realize you can use just `-h`. There is also `--version` or `-V`. I'll incorporate those.